### PR TITLE
Adding new breadcrumb file.

### PR DESCRIPTION
**Pull request by trackleft**

### CODE REVIEW

**Comment by trackleft** [comment]
I didn't mean to change any of the styles in my Pull request, just the file structure.

File: scss/_breadcrumb.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.breadcrumb {
```
Comment (trackleft) [inline scss/_breadcrumb.scss:1]:
```suggestion
// Overrides for Breadcrumbs
.breadcrumb {
```

File: scss/_breadcrumb.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      &::before {
+        float: left;// Suppress inline spacings and underlining of the separator
+        padding-right: $breadcrumb-item-padding;
+        // stylelint-disable-next-line
+        font-family: "Material Icons Sharp";
+        color: $breadcrumb-divider-color;
+        content: "chevron_right";
+      }
+    }
+  }
```
Comment (cpoff) [inline scss/_breadcrumb.scss:34]: We seem to be missing a closing bracket?

File: scss/_breadcrumb.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (trackleft) [inline scss/_breadcrumb.scss:34]:
```suggestion
    }
  }
```

**Review by cpoff — APPROVED** [review]

**Comment by cpoff** [comment]
This has been integrated in the original PR. Thanks Chris!

**Pull request closed without merging**